src/lib/LUA/rx_devLUA.cpp

```diff
@@ -627,6 +667,21 @@ static int event()
   }
   setLuaTextSelectionValue(&luaBindStorage, config.GetBindStorage());
   updateBindModeLabel();
+
+  if (config.GetSourceSysId()==0) 
```

Comment:
(nitpick) can we add some whitespace between the equality op here? E.g.
`if (config.GetSourceSysId() == 0)`